files/autostop/stop.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+. /start-utils
+
+logAutostopAction "Stopping Java process"
+
+rcon-cli stop >/dev/null
```

Comment:
Some users may have rcon disabled. Even though that could be documented as a requirement, I'm thinking it would be more robust to send a TERM signal to pid 1, which is what docker stop is doing.

Reply:
done, rcon is even invoked when it is enabled by the server itself